src/StdUtils.sol

```diff
@@ -116,6 +117,24 @@ abstract contract StdUtils {
         return addressFromLast20Bytes(keccak256(abi.encodePacked(bytes1(0xff), deployer, salt, initcodeHash)));
     }
 
+    /// @dev returns the address of a contract created with CREATE2 using the default CREATE2 deployer
+    function computeCreate2Address(bytes32 salt, bytes32 _initCodeHash) internal pure returns (address) {
+        return computeCreate2Address(salt, _initCodeHash, DEFAULT_CREATE2_DEPLOYER);
+    }
+
+    /// @dev returns the hash of the init code (creation code + no args) used in CREATE2 with no constructor arguments
+    /// @param creationCode the creation code of a contract C, as returned by type(C).creationCode
+    function hashInitCodeNoConstructorArgs(bytes memory creationCode) public pure returns (bytes32) {
```

Comment:
Thoughts on just calling this `hashInitCode` and overloading that method? Main concern of that change is that it may lead to confusion for users who aren't too familiar with how to get the initcode hash for the contracts, whereas the current name is more explicit, but that could be handled by docs in the foundry book 🤔
```suggestion
    function hashInitCode(bytes memory creationCode) public pure returns (bytes32) {
```

Reply:
makes sense, it was indeed on the wordy side